style.css

```diff
@@ -0,0 +1,107 @@
+* {
+    margin: 0;
+}
+html{
+    scroll-behavior: smooth;
+}
+body{
+    background-color: #29058f;
+    min-height: 100vh;
+    color: #abcdef;
+}
+.navbar{
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    background-color: #111111f2;
+    padding: 20px;
+    z-index: 100;
+}
+.name{
+    flex: 0.6;
+    font-size: 22px;
+}
+.link{
+    flex: 0.4;
+    font-size: 22px;
+}
+a{
+    text-decoration: none;
+    color: #fff;
+    padding: 20px;
+}
+a:hover{
+    background-color: #333;
+}
+@media (max-width: 1000px) {
+    .namw{

```

Comment:
on this line of CSS, I think you meant to type `name` instead of `namw`. I would recommend correcting that issue, so the HTML and CSS can work together 👍